mxcube3/ui/components/RemoteAccess/UserList.jsx

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import { bindActionCreators } from 'redux';
 import { connect } from 'react-redux';
 import { Button, Panel } from 'react-bootstrap';
-import { sendGiveControl } from '../../actions/remoteAccess';
+import { sendGiveControl, sendLogoutUser } from '../../actions/remoteAccess';
 
 class UserList extends React.Component {
   getObservers() {
```

Comment:
would be good to include the one in control too

Reply:
I don't think I understood this one ?, do you want to force logout the user in_control ?

Reply:
aah no, just list the one in control in the User list